index.js

```diff
@@ -160,7 +162,7 @@ export default class ModalSelector extends BaseComponent {
                             </View>
                         </ScrollView>
                     </View>
-                    <View style={styles.cancelContainer}>
+                    <View style={[styles.cancelContainer, this.props.cancelContainerStyle]}>
```

Comment:
This will mix `props.cancelContainerStyle` with `styles.cancelContainer`, preventing the caller from eliminating the `maxHeight` style.  It could be overridden, but a value would have to be chosen instead of allowing RN's layout manager to size to the contents.

To allow props.cancelContainerStyle to replace the built-in style, this could be done:
```
<View style={[this.props.cancelContainerStyle || styles.cancelContainer]}>
```

Reply:
If `maxHeight` and `flexGrow` is removed from `cancelContainer` then I think the comma-separated is ok. And I'm also going towards the comma-separated anyways, for consistency reasons. But I do see your point in eliminating `cancelContainer`.

Reply:
My stance is that the library should work for most cases out of the box.  That means maxHeight & flexGrow would need to be there by default.
However if the user finds that doesn't work for them, they should be able to completely override (and remove) those styles with their own.

Reply:
I totally agree that the library should be usable for most people out of the box, but I did some testing with removing both `maxHeight` and `flexGrow`, and that seemed actually to be a better default. 

iOS with `maxHeight` and `flexGrow` removed from `cancelContainer`:
![ios_selector2](https://user-images.githubusercontent.com/2505178/33286822-de07187a-d3b6-11e7-805c-36a1dba94c29.png)

Android (some _with_ `maxHeight` and `flexGrow` and some _without_):
![android_selector](https://user-images.githubusercontent.com/2505178/33287169-1a02053c-d3b8-11e7-9916-71051ac2c8e3.png)